setup.py

```diff
@@ -4,12 +4,15 @@
 import os
 from setuptools import setup, find_packages
 
+import formidable
+
+
 with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
     README = readme.read()
 
 setup(
     name='django-formidable',
-    version='0.6.0.dev0',
+    version=formidable.version,
```

Comment:
I saw this pattern several times, so I think it's safe.
Alternatively, we could use [bumpversion package](https://pypi.python.org/pypi/bumpversion) to dump the new version in both files.